src/main/webapp/i18n/de/global.json

```
@@ -24,12 +24,12 @@
             "entities": {
                 "main": "Entitäten",
                 "course": "Kurse",
-                "exercise": "Exercise",
-                "textExercise": "Text Exercise",
-                "fileUploadExercise": "File Upload Exercise",
-                "programmingExercise": "Programming Exercise",
-                "modelingExercise": "Modeling Exercise",
-                "quizExercise": "Quiz Exercise",
+                "exercise": "Aufgabe",
+                "textExercise": "Text Aufgabe",
+                "fileUploadExercise": "File Upload Aufgabe",
+                "programmingExercise": "Programming Aufgabe",
+                "modelingExercise": "Modeling Aufgabe",
+                "quizExercise": "Quiz Aufgabe",
```

As far as I am aware we are using other translations here

```suggestion
                "exercise": "Aufgabe",
                "textExercise": "Textaufgabe",
                "fileUploadExercise": "Datei-Upload-Aufgabe",
                "programmingExercise": "Programmieraufgabe",
                "modelingExercise": "Moderllierungsaufgabe",
                "quizExercise": "Quiz Aufgabe",
```

---

I removed these translations completely, as far as I can see, we don't use them anyway.